cli/print/customers.go

```diff
@@ -14,9 +15,23 @@ var customersTmplSrc = `ID	NAME	CHANNELS	EXPIRES	TYPE
 
 var customersTmpl = template.Must(template.New("channels").Parse(customersTmplSrc))
 
-func Customers(w *tabwriter.Writer, customers []types.Customer) error {
-	if err := customersTmpl.Execute(w, customers); err != nil {
+func Customers(outputFormat string, w *tabwriter.Writer, customers []types.Customer) error {
+	if outputFormat == "table" {
+		if err := customersTmpl.Execute(w, customers); err != nil {
+			return err
+		}
+		return w.Flush()
+	}
+	if outputFormat == "json" {
+		defer w.Flush()
+		var cAsByte []byte
+		if len(customers) == 1 {
+			cAsByte, _ = json.MarshalIndent(customers[0], "", "  ")
+		} else {
+			cAsByte, _ = json.MarshalIndent(customers, "", "  ")
+		}
```

Comment:
I'd recommend against this sort of choice - this means that the output type is different depending on if there's 1 customer (hashmap), or >1 customer (array)... which makes it harder to write a script or output parser that can handle both scenarios.

Reply:
So always return it as an array?

Reply:
That would be my preference, yes. :)

Reply:
Nevermind. We discussed this. I'll find a better way.

Reply:
Adding summary from Zoom call - this function is called for both `customer get $CUSTOMER` and `customer ls` type commands, which would be a desirable use case for having two separate output styles (it makes sense for a "get" type request to output the single customer as a hashmap, and the "ls" style request to output an array always).

Reply:
Better: It is used for both `customer create` and `customer ls`. There is no `customer get` command at the moment.
I've split it into 2 different print functions.